website/docs/r/resource.html.markdown

```diff
@@ -35,14 +35,46 @@ EOT
 }
 ```
 
+###Using full_data
+
+```hcl
+resource "turbot_resource" "my_resource" {
+  parent   = "tmod:@turbot/turbot#/"
+  type     = "tmod:@turbot/aws#/resource/types/account"
+  full_data     = <<EOT
+{
+  "foo": "bar",
+  "title": "turbot account resource"
+}
+EOT
+}
+```
+
+###Using full_metadata
+
+```hcl
+resource "turbot_resource" "my_resource" {
+  parent   = "tmod:@turbot/turbot#/"
+  type     = "tmod:@turbot/aws#/resource/types/account"
+  full_metadata    = <<EOT
+{
+  "resource_version": "1.0.0"
+  "replication": "true"
+}
+EOT
+}
+```
+
 ## Argument Reference
 
 The following arguments are supported:
 
 - `parent` - (Required) The `id` or `aka` of the level at which the Turbot resource will be created.
```

Comment:
- `parent` - (Required) The identifier of the parent resource under which this resource will be created.